packages/catalog-generator/src/test/java/io/kaoto/camelcatalog/MainTest.java

```diff
@@ -0,0 +1,13 @@
+package io.kaoto.camelcatalog;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+
+public class MainTest {
+
+    @Test
+    public void testGenerate() throws IOException {
+        // Main.main(new String[] {});
```

Comment:
Should we uncomment?

Reply:
@lordrip ?

Reply:
We need to bring and adapt the tests from the `@kaoto/camel-catalog` project, we can uncomment, but nevertheless, the entire file will be replaced.